### PR TITLE
day4

### PR DESCRIPTION
**Pull request by KhasanMeliev**

### CODE REVIEW

File: Day4/day4.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+//   let array = [1, 2, 3, 4, 5];
+//   array.splice(index, 0, elem);
+//   return array;
+// }
+// console.log(insertElement(8, 3));
+
+//--------------------------------Compare two arrays---------------------------------
+// const array1 = [1, 3, 5, 8];
+// const array2 = [1, 3, 5, 8];
+// function compareArrays(arr1, arr2) {
```
Comment (javokhirm) [inline Day4/day4.js:26]: Try to use another algorithm for this problem

File: Day4/day4.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+// } else {
+//   console.log("key doesn't exists");
+// }
+
+//------------------------- Add Key/Value Pair to an Object---------------
+// let user = {
+//   name: "hasan",
+//   age: 13,
+// };
+// let add = (user["hobby"] = "coding");
```
Comment (javokhirm) [inline Day4/day4.js:80]: Why we need let add = ?

**Review by javokhirm — CHANGES_REQUESTED** [review]
Fix problems according to comments

**Comment by KhasanMeliev** [comment]
Add qowayabman key/value wuning ucun wunaqa yozgandim

**Pull request merged**